api/cluster/templater.go

```diff
@@ -347,6 +321,50 @@ func (t *KFServingResourceTemplater) createTransformerSpec(modelService *models.
 	return transformerSpec
 }
 
+func (t *KFServingResourceTemplater) enrichStandardTransformerEnvVars(envVars models.EnvVars) models.EnvVars {
```

Comment:
In this method there are 2 variables with almost same name (`standardTransformerConfig` and `t.standardTransformerConfig`) that has significantly different meaning. Can you change one of it.